### PR TITLE
mkhomedir_helper: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/mkhomedir_helper.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# mkhomedir_helper
+
+> Create the user home folder after creating the user.
+> More information: <https://manned.org/mkhomedir_helper.8>.
```
Comment (pixelcmtd) [inline pages/linux/mkhomedir_helper.md:4]:
```suggestion
> More information: <https://manned.org/mkhomedir_helper>.
```

**Review by pixelcmtd — COMMENTED** [review]
Could you please add an example using `umask`, `path-to-skel` and `home_mode`?

**Comment by Managor** [comment]
I have no idea what `home_mode` does so I dare not to touch that.

**Review by pixelcmtd — APPROVED** [review]

File: pages/linux/mkhomedir_helper.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# mkhomedir_helper
+
+> Create the user home folder after creating the user.
+> More information: <https://manned.org/mkhomedir_helper>.
+
+- Create a home folder for a user based on /etc/skel with umask 022:
+
+`sudo mkhomedir_helper {{user_name}}`
```
Comment (acuteenvy) [inline pages/linux/mkhomedir_helper.md:8]:
```suggestion
- Create a home directory for a user based on `/etc/skel` with umask 022:

`sudo mkhomedir_helper {{username}}`
```

File: pages/linux/mkhomedir_helper.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Create a home folder for a user based on /etc/skel with all permissions for owner (0) and read permission for group (3):
+
+`sudo mkhomedir_helper {{user_name}} {{037}}`
```
Comment (acuteenvy) [inline pages/linux/mkhomedir_helper.md:12]:
```suggestion
- Create a home directory for a user based on `/etc/skel` with all permissions for owner (0) and read permission for group (3):

`sudo mkhomedir_helper {{username}} {{037}}`
```

File: pages/linux/mkhomedir_helper.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# mkhomedir_helper
+
+> Create the user home folder after creating the user.
+> More information: <https://manned.org/mkhomedir_helper>.
```
Comment (acuteenvy) [inline pages/linux/mkhomedir_helper.md:4]:
```suggestion
> Create the user's home directory after creating the user.
> More information: <https://manned.org/mkhomedir_helper>.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**